javascript/code-challenges/quickSort/BLOG.md

```diff
@@ -0,0 +1,3 @@
+# Quick Sort
+
+To quickSort,  pass the array, left-pointer and right-pointer
```

Comment:
With everything going on, take your time coming back to this and I will regrade when you are ready!